### DISCUSSION TITLE
Hi, i'm new here and found a little mistake in one of your php example files...

### DISCUSSION THREAD

**Discussion by iLoZ79** [community/community#143006] [New to GitHub] [answered]
https://github.com/RobinNixon/lpmj6/blob/master/05/16.php <-- in this file there is a '}' missing before closing the script with ?>... Maybe someone can correct is. Thanks!

**Comment by ebndev** [comment, marked as answer]
Hi @iLoZ79, thanks for being a part of the GitHub Community!

You might want to try [opening a PR](https://github.com/RobinNixon/lpmj6/pulls) in their repo. 

GitHub Community Discussions is a public forum and focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for the project owners to see your message.

Best of luck!
